docs/how-to-guides/environment-setup-and-maintenance/installing-isis-via-anaconda.md

```diff
@@ -139,9 +143,9 @@ ISIS requires these environment variables to be set in order to run correctly:
             # conda config vars set KEY=VALUE
             ```
 
-        1.  This command sets both required variables (fill in your `ISISDATA` path):
+        1.  This command sets both required variables and the PATH (fill in your `ISISDATA` location):
 
-                conda env config vars set ISISROOT=$CONDA_PREFIX ISISDATA=[your data path]
+                conda env config vars set ISISROOT=$CONDA_PREFIX ISISDATA=[your data path] PATH=$PATH:$CONDA_PREFIX/bin
```

Comment:
e.g., here, this would not accomplish much since `$CONDA_PREFIX/bin` is already in your PATH on activation.